ansible/roles/automation_controller_issue_cert/files/lets-encrypt-x3-cross-signed.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Urgh, I don't think there should be a key in the repository, it should actually be ignored in .gitignore.

Reply:
Or is it the public key of Let's Encrypt?

Reply:
It's the cross signed intermediate, this is from the Ansible WS provisioner role. Just noticed on https://letsencrypt.org/certificates/ it's actually retired, sigh. Still works, though.